docs/control-structures.rst

```diff
@@ -60,31 +63,73 @@ Input parameters and output parameters can be used as expressions in
 the function body.  There, they are also usable in the left-hand side
 of assignment.
 
+
+.. _multi-return:
+
+Returning Multiple Values
+-------------------------
+
+When a function has multiple output parameters, ``return (v0, v1, ...,
```

Comment:
Wouldn't it be better to just move that up where it is linked from?
"The ``return`` statements are also capable of returning multiple values using ``return (v1, v2, ..., vn)``. The number of components have to be the same as the number of output parameters.'

Reply:
@chriseth Ahh, good idea, didn't see that before.